srcpkgs/calibre/template

```diff
@@ -52,9 +53,11 @@ do_check() {
 	# qt test crashes without any error messages
 	# disabled for now
 	# unrar test fails because python3-unrardll isn't packaged
+	# searching test case fails comparing old backend with new
 	python3 setup.py test \
 		--exclude-test-name unrar \
-		--exclude-test-name qt
+		--exclude-test-name qt \
+		--exclude-test-name searching
```

Comment:
If this is a new test, it might still have one issue or another. Will investigate

Reply:
seems like an old test.. latest changes were done more than a year ago.. :\

Reply:
Ok, test works on 5.4.2. This is interesting.

Reply:
weird.. i get the same on 5.4.2..

Reply:
Oh nice, this is a locale generated bug. Do you have an account in their bug tracker, by any chance?

Reply:
no :S

Reply:
Ok, this test works on my machine. So please leave a comment that failure isn't deterministic.